docs/library/urandom.rst

```diff
@@ -0,0 +1,58 @@
+:mode:`urandom` -- generates random numbers
+===========================================
+
+.. module:: urandom
+   :synopsis: random numbers
+
+This module implements pseudo-random number generator.
+
+Given below are the methods that this module provides.
+
+ 
+  .. method:: urandom.getrandbits(n)
+
+    Returns an integer with 'n' random bits (1<=n<=32).
+    
+  .. method:: urandom.choice(sequence)
+   
+    Chooses multiple random items from a sequence (list, set or any
+    data structure).
+    
+  .. method:: urandom.seed(n=None, /)
+  
+    Initialises random number generator module with n. The None case only 
+    works if :MICROPY_PY_URANDOM_SEED_INIT_FUNC: is defined in the port, 
+    otherwise raises ValueError.
+
+  .. method:: urandom.randint(a, b)
+    
+    Returns a random interger in the range [a, b].
+    
+  .. method:: urandom.randrange(stop)
+  
+    Returns a random number from the range [0, stop).
+
+  .. method:: urandom.randrange(start, stop)
```

Comment:
All 3 of these randrange definitions should be combined into one, like Python: https://docs.python.org/3/library/random.html?highlight=randrange#random.randrange

Reply:
like this:
```
.. method:: randrange(stop)
            randrange(start, stop[, step])

    <description of the various ways to call this function>
```

Reply:
and this function should go in the integer section